documentation/modules/creating-storage-mapping.adoc

```diff
@@ -23,14 +23,14 @@ You can create a storage mapping by using the {ocp} web console to map source di
 * *Source provider*: Select a source provider.
 * *Target provider*: Select a target provider.
 
-. Map source disk storages to target storage classes as follows:
+. To create a storage mapping, click *Add* and map source disk storages to target storage classes as follows:
```

Comment:
```suggestion
. To create a storage mapping, click *Add* and map source disk storage to target storage classes as follows:
```

Unsure about this one… I think storage might convey singular or plural

Reply:
yeah, github also doesn't like `storages`..
how about:
```suggestion
. To create a storage mapping, click *Add* and map disk storage sources to target storage classes as follows:
```